network-api/networkapi/templates/fragments/research_library_form.html

```diff
@@ -1,14 +1,16 @@
 {% load l10n %}
 {% for field in form %}
-    <div class="tw-pt-8 tw-pb-24 tw-border-t">
-        <fieldset>
-            <legend>
-                <h3 class="tw-h4-heading">{{ field.label_tag }}</h3>
-            </legend>
-            {{ field.errors }}
-            <ul class="tw-list-none tw-mb-0 tw-pl-0 {% if field|length > 5 %} tw-max-h-[13.5rem] tw-overflow-y-scroll{% endif %}">
-                {% for option in field %}<li class="tw-flex tw-flex-row tw-items-baseline">{{ option }}</li>{% endfor %}
-            </ul>
-        </fieldset>
-    </div>
+    {% if field|length %}
```

Comment:
Thanks @jhonatan-lopes!!

I actually tried to fix this myself the other day and got stuck as `field.choices`, `field.length` and all my other attempts were not working. 

Cool to see that `|length` works!

Reply:
@danielfmiranda Yep, this took me a while to figure out as well, but there is a cool example on the official docs that does exactly that.